docs/develop/get-started-with-ton.mdx

```diff
@@ -18,7 +18,7 @@ This learning path contains __5 modules__ and should take you around __45 minute
 
 In this tutorial, you'll learn how to easily make blockchain transactions using JavaScript. You could learn to do it without this tutorial, but this way is cool and accessible.
 
-1. You will make your own TON Wallet with Tonkeeper
+1. You will make your own TON Wallet with TonKeeper
```

Comment:
Tonkeeper is a correct branding, let's check on their website (https://tonkeeper.com/)